src/puzzle/Graphic.js

```diff
@@ -480,7 +480,12 @@
 						x1 >= bd.maxbx + bm ||
 						y1 >= bd.maxby + bm ||
 						x2 <= bd.minbx - bm ||
-						y2 <= bd.minby - (bm + (this.pid === "starbattle" ? 2 : 0)))
+						y2 <=
+							bd.minby -
+								(bm +
+									(this.pid === "starbattle" || this.pid === "isowatari"
```

Comment:
I think this looks pretty bad after the formatting, could you maybe add this line and reference it from this if statement?
```js
var hasIndicator = this.pid === "starbattle" || this.pid === "isowatari";
```